prometheus/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - Prometheus
 
+## 3.0.0 / 2018-10-12
+
+* [Changed] Change default prometheus metric limit to 2000. See [#2248](https://github.com/DataDog/integrations-core/pull/2248).
```

Comment:
Should this be released? Is this a PR and then a revert of that PR?

Reply:
No, this is the first released default prometheus metric limit change.